gamedata/movedefs.lua

```diff
@@ -293,8 +293,8 @@ local moveDatas = {
 	-- cormlv armmflash corgator legmrv armmlv leghades leghelops armfav corfav armconsul armlatnk cortorch legmrrv
 	TANK2 = {
 		crushstrength = 18,
-		footprintx = 2,
-		footprintz = 2,
+		footprintx = 3,
+		footprintz = 3,
```

Comment:
Shouldn't this be called TANK3 now? It seemed the name was intended to match the foorptinr size,